learning/tour-of-beam/frontend/lib/pages/main/screen.dart

```diff
@@ -0,0 +1,304 @@
+import 'package:playground_components/theme/color_provider.dart';
+import 'package:playground_components/widgets/split_view.dart';
+
+import '../../components/borderless_expandable_wrapper.dart';
+import '../../components/complexity.dart';
+import '../../components/page_container.dart';
+import '../../constants/sizes.dart';
+import '../../generated/assets.gen.dart';
+
+class MainScreen extends StatelessWidget {
+  const MainScreen();
+
+  @override
+  Widget build(BuildContext context) {
+    return PageContainer(
+      content: Row(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          const _Summary(),
+          Flexible(
+            child: SplitView(
+              themeData: Theme.of(context),
+              first: const _Content(),
+              second: const _Playground(),
+              direction: SplitViewDirection.horizontal,
+            ),
+          ),
+        ],
+      ),
+    );
+  }
+}
+
+class _Summary extends StatelessWidget {
+  const _Summary();
+
+  @override
+  Widget build(BuildContext context) {
+    return Container(
+      width: 250,
+      padding: const EdgeInsets.symmetric(horizontal: BeamSizes.size12),
+      child: SingleChildScrollView(
+        child: Column(
+          children: const [
+            SizedBox(height: BeamSizes.size12),
+            _SummaryTitle(),
+            _Modules(),
+            SizedBox(height: BeamSizes.size12),
+          ],
+        ),
+      ),
+    );
+  }
+}
+
+class _Modules extends StatelessWidget {
+  const _Modules();
+
+  @override
+  Widget build(BuildContext context) {
+    return Padding(
+      padding: const EdgeInsets.only(top: BeamSizes.size16),
+      child: Column(
+        children: const [
+          _ModuleTitle(),
+          _Groups(),
+          _Groups(),
+        ],
+      ),
+    );
+  }
+}
+
+class _SummaryTitle extends StatelessWidget {
+  const _SummaryTitle();
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Text(
+          'pages.main.summaryTitle',
+          style: Theme.of(context).textTheme.headlineLarge,
+        ).tr(),
+        SvgPicture.asset(Assets.svg.summaryHideArrow),
+      ],
+    );
+  }
+}
+
+class _ModuleTitle extends StatelessWidget {
+  const _ModuleTitle();
+
+  @override
+  Widget build(BuildContext context) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceBetween,
+      children: [
+        Text(
+          'Core Transforms',
+          style: Theme.of(context).textTheme.headlineMedium,
+        ),
+        const Padding(
+          padding: EdgeInsets.only(right: BeamSizes.size4),
+          child: ComplexityWidget(complexity: Complexity.basic),
+        ),
+      ],
+    );
+  }
+}
+
+class _Groups extends StatelessWidget {
+  const _Groups();
+
+  @override
+  Widget build(BuildContext context) {
+    return const BorderlessExpandableWrapper(
```

Comment:
Horizontal padding is insufficient.
Edges need to be rounded.

![image](https://user-images.githubusercontent.com/44893228/186705545-87461cd1-aff9-472e-8627-ade3a3e40a40.png)
![image](https://user-images.githubusercontent.com/44893228/186705633-489274b1-4b5f-4312-9f35-b7c51d4f896a.png)

Reply:
Yes, I will go through the layout details after I get rid of expandable's vertical padding.

Reply:
I also need to make the Table of Contents closable.

Reply:
Decided to leave it always open.